src/dbnode/persist/fs/commitlog/commit_log_test.go

```diff
@@ -437,7 +437,7 @@ func TestReadCommitLogMissingMetadata(t *testing.T) {
 
 func TestCommitLogReaderIsNotReusable(t *testing.T) {
 	// Make sure we're not leaking goroutines
-	defer leaktest.CheckTimeout(t, time.Second)()
+	defer leaktest.CheckTimeout(t, 10*time.Second)()
```

Comment:
Not sure if bumping this is necessary, you might just need to rebase master. Prateek upgraded to the latest version of leaktest which fixes this